diff --git a/src/options/opt.h b/src/options/opt.h
index fca2bbf8..b8bbfadf 100644
--- a/src/options/opt.h
+++ b/src/options/opt.h
@@ -77,7 +77,7 @@ const uint32_t NOEOF = ~0u - 1;
 #define RE2C_CONSTOPTS \
     CONSTOPT1 (target_t, target, TARGET_CODE) \
     CONSTOPT (lang_t, lang, RE2C_LANG) \
-    CONSTOPT (bool, bNoGenerationDate, false) \
+    CONSTOPT (bool, bNoGenerationDate, true) \
     CONSTOPT (bool, version, true) \
     CONSTOPT (bool, cFlag, false) \
     CONSTOPT (bool, fFlag, false) \
